lib/ansible/galaxy/collection/__init__.py

```diff
@@ -997,7 +997,7 @@ def find_existing_collections(path, artifacts_manager):
 
             display.vvv(
                 "Found installed collection {coll!s} at '{path!s}'".
-                format(coll=to_text(req), path=to_text(req.src))
+                format(coll=to_native(req), path=to_native(req.src))
```

Comment:
Display should take in text strings not not native strings. We usually leave native for exceptions though.

Reply:
Right, so maybe having a u-string should be fine.

Reply:
Yep, they were incorporated into https://github.com/webknjaz/ansible/pull/357.